public/views/signup.html

```diff
@@ -1,85 +1,117 @@
+                    <label for="email" class="control-label"></label>
+                    <div class="controls">
+                        <input type="email" placeholder="Email" ng-model="user.email">
+                    </div>
+                </div>
+                <p ng-show="validationErrorExists()" class="error-alert">{{ validationErrorMessage[1] }}</p>
+                <p ng-show="validationErrorExists()" class="error-alert">{{ validationErrorMessage[3] }}</p>
+              <div class="control-group">
+                  <label for="password" class="control-label pass"></label>
+                  <div class="controls">
+                    <input type="password" placeholder="Password" ng-model="user.password">
+                  </div>
               </div>
-            </div>
-            <div class="control-group">
-              <label for="password" class="control-label"></label>
-              <div class="controls">
-                <input class="input-info" id="password" type="password" name="password" placeholder="Password" ng-model="user.password" required novalidate />
+              <p ng-show="validationErrorExists()" class="error-alert">{{ validationErrorMessage[2] }}</p>
+              <div class="control-group">
+                <div class="controls">
+              <button type="submit" id="login-button" ng-click="signup(user)">SIGN UP</button>
+                </div>
               </div>
-            </div>
-            <div id="contain-avatars">
-              <ul ng-controller="IndexController" class="avatar-list">
-                <li ng-repeat="avatar in avatars" class="avatars ng-scope">
-                  <input name="avatar" type='radio' value="{{$index}}">
-                    <img ng-src="{{avatar}}">
-                  </input>
-                </li>
-              </ul>
-            </div>
-            <div class="form-actions">
-              <button id="sign-up-btn-other" type='submit' ng-click="signup(user)">Sign Up</button>
-            </div>
-          </form>
+            </form>
+          </div>
+          
+          <ul class="bg-bubbles">
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+          </ul>
         </div>
       </div>
-    </div>
-    </div>
-    </div>
-  </div>
-</div>
-
-
-
+      <footer class="footer fixed-bottom">
+        <p class="text-center">&copy 2018 CARDS FOR HUMANITY Inc</p>
+      </footer>
+      </section>
+      <script src="vendor/jquery/jquery.min.js"></script>
+      
+          <!-- Bootstrap Core JavaScript -->
+          <script src="vendor/bootstrap/js/bootstrap.min.js"></script>
+      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
Installing Bootstrap through bower automatically installs JQuery. Why do we need a cdn again?

Reply:
Thanks for pointing that out, I'll make changes to the affected files ASAP.